app/src/main/java/ch/sdp/vibester/activity/IncorrectSongsActivity.kt

```diff
@@ -1,62 +1,63 @@
 package ch.sdp.vibester.activity
 
+import android.content.Intent
 import android.os.Bundle
 import android.view.Gravity
-import android.widget.Button
+import android.view.View
 import android.widget.LinearLayout
 import android.widget.LinearLayout.LayoutParams
 import android.widget.TextView
 import androidx.appcompat.app.AppCompatActivity
 import ch.sdp.vibester.R
-import ch.sdp.vibester.helper.IntentSwitcher
 
```

Comment:
maybe can add documentation here ? What does the class represent ? maybe method ?

Reply:
Done in the recent commit. Thanks for the feedback!